docs/integrations-in-rancher/cloud-marketplace/azure-marketplace-payg-integration/installing-rancher-prime.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Installing Rancher Prime PAYG on Azure
+---
+
+This page covers how to install the Rancher Prime PAYG offering on Microsoft's Azure Marketplace.
+
+## How to Install Rancher Prime PAYG
+
+Refer to the following steps for creating a new deployment of Rancher Prime from the Azure Marketplace page.
+
+1. Select the **Rancher Prime with 24x7 Support** offer (either **EU and UK only** or **non-EU and non-UK only**) that corresponds to the location where your account is registered.
```

Comment:
The offer titles have changed since the original publishing and now are "Rancher Prime with 24x7 Support (EMEA Orders Only)" and "Rancher Prime with 24x7 Support"
The text in the doc here is accurate for what we originally published, but should be updated to reflect the changes in the listings